### DISCUSSION TITLE
Separate upfront and production carbon intensities

### DISCUSSION THREAD

**Discussion by veqtrus** [electricitymaps/electricitymaps-contrib#2842] [Ideas]
Currently ElectricityMap assumes that carbon emissions are directly proportional to current production which is not that accurate. Particularly it shows that emissions are zero when there is no production. The emissions from non-fossil sources mostly come from building the power plants, rather than during production of electricity.

One effect is that the website shows that emissions are increasing when wind or solar production increases, whereas in fact the emissions from increased production shouldn't increase a lot. On the other hand if a source has a very low capacity factor in a particular region (e.g. a solar plant in a region where it's typically cloudy) the upfront carbon emissions aren't properly attributed.

Hence I would suggest calculating carbon emissions using a formula like `upfront_intensity * installed_capacity + production_intensity * current_production`. I suppose we could find the values of these parameters if we dig into the studies that estimated carbon intensities.

**Comment by corradio** [comment]
Hi @veqtrus,

Ecoinvent has some of these values. In the paper that we published [here](https://arxiv.org/pdf/1812.06679.pdf), you will find some tables that separate operational intensities.

**Reply by veqtrus** [reply]
Thank you for this reference. Some of your values differ considerably from the IPCC medians, especially solar. Is there some explanation?

**Reply by corradio** [reply]
I don't remember the specifics but I suggest you check out this long thread about the topic:
https://github.com/tmrowco/electricitymap-contrib/issues/738

Hopefully we can get to work on this sooner than later
